src/components/relatedItems/ComparisonModal.jsx

```diff
@@ -23,9 +23,9 @@ const ComparisonModal = (props) => {
 
       return (
         <tr>
-          <td>{currValue !== undefined ? currValue.value : 'N/A'}</td>
+          <td>{!!currValue ? currValue.value : 'N/A'}</td>
```

Comment:
Wait is  !!currValue the same as currValue?

Reply:
oh yeah lol I forgot I left that in there! I was trying to catch one of the features in the comparison table that I think is null, but for some reason is not triggering the second condition in the ternary. It's redundant, but I was basically trying to ensure that currValue is being coerced to a boolean (i.e., to explicitly coerce null to false and set the value to N/A).

Reply:
ahhh that makes sense now haha